docs/options/image.md

```
@@ -0,0 +1,39 @@
+|  ROCm (AMD)             | quay.io/ramalama/rocm      |
+|  CUDA (NVIDIA)          | quay.io/ramalama/cuda      |
+|  Intel GPU (sycl)       | quay.io/ramalama/intel-gpu |
+|  Intel GPU (openvino)   | ghcr.io/ggml-org/llama.cpp:full-openvino |
+|  Asahi (Apple Silicon)  | quay.io/ramalama/asahi     |
+|  CANN (Ascend)          | quay.io/ramalama/cann      |
+|  MUSA (Moore Threads)   | quay.io/ramalama/musa      |
+
+Upstream llama.cpp "full" images from `ghcr.io/ggml-org/llama.cpp` are also supported.
+RamaLama automatically detects the image type and adjusts the container CLI accordingly.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)